### ISSUE TITLE
The FlutterMap onTap not work when using this package

### ISSUE THREAD

**Issue by jiazeh** [OwnWeb/flutter_map_tappable_polyline#18]
The code indicated below does not work(see: _handleTap)
```
  FlutterMap(
        options: MapOptions(
          onTap: _handleTap, ///*** here not work *** 
          plugins: [
            TappablePolylineMapPlugin(),
          ],
          center: LatLng(45.1313258, 5.5171205),
          zoom: 11.0,
        ),
        layers: [
          TileLayerOptions(
            urlTemplate: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
            subdomains: ['a', 'b', 'c'],
          ),
          TappablePolylineLayerOptions(
              // Will only render visible polylines, increasing performance
              polylineCulling: true,
              pointerDistanceTolerance: 20,
              polylines: [
                TaggedPolyline(
                  tag: 'My Polyline',
                  // An optional tag to distinguish polylines in callback
                  points: getPoints(),
                  color: Colors.red,
                  strokeWidth: 3.0,
                ),
              ],
              onTap: (TaggedPolyline polyline) => print(polyline.tag),
              onMiss: () {
                print('No polyline was tapped');
              })
        ],
      ),

  /// _handleTap not work
  void _handleTap(LatLng latlng) {
    setState(() {
      tappedPoints.add(latlng);
    });
  }

```

**Comment by tuarrep** [comment]
Could you try pulling this PR to test? 
https://github.com/OwnWeb/flutter_map_tappable_polyline/pull/16

It should fix this issue. 
We're waiting for internal testing before the merge. I hope @HugoHeneault will have a quick look this week

**Comment by jiazeh** [comment]
@tuarrep Thanks a lot, it work.

**Comment by aliciasteiman** [comment]
Has the PR been merged yet? I'm having the same issue... Thanks!

**Comment by tuarrep** [comment]
Merge is in progress

**Comment by Zeeshan0201** [comment]
any update on this?

**Comment by HugoHeneault** [comment]
Hello @Zeeshan0201. This issue is closed as #16 has been merged. Are you experiencing issues?
